packages/sentry/src/sentry.ts

```diff
@@ -32,13 +37,18 @@ export const updateUser = user => {
 };
 
 export const stopTrackingUser = () => {
-  if (window.app.trackingSentry)
+  if (window.app.trackingSentry) {
     Sentry.configureScope(scope => {
       scope.clear();
     });
+  }
 };
 
-export const reportErrorToSentry = (error, extraInfo, getIsEnabled) => {
+export const reportErrorToSentry = (
```

Comment:
shouldn't this have a return type as well?

Reply:
Most of the time, type are inferred, so we don't need to explicitly annotate them.

Reply:
Okay as long as that's the best practice. We should be careful though with our "first typescript package", as we will use it as inspiration for the rest of the typescript we write going forward...

Reply:
Sure...you will see it yourself though when you write code that if something needs to be annotated, you'll get a compiler warning/error. If not, you just leave the annotation out.